modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -67,43 +68,47 @@ If you want to create virtual function on the default network interface, ensure
 
 <8> The number of the virtual functions (VF) to create for the SR-IOV physical network device. For an Intel network interface controller (NIC), the number of VFs cannot be larger than the total VFs supported by the device. For a Mellanox NIC, the number of VFs cannot be larger than `128`.
 
-<9> The NIC selector identifies the device for the Operator to configure. You do not have to specify values for all the parameters. It is recommended to identify the network device with enough precision to avoid selecting a device unintentionally.
+<9> Set `externallyManaged` to `true` to allow the SR-IOV Network Operator to use all or a subset of externally managed virtual functions (VFs) and attach them to pods. With the value set to `false` the SR-IOV Network Operator manages and configures all allocated VFs.
++
+[NOTE]
+====
+When `externallyManaged` is set to `true`, you must create the Virtual Functions (VFs) before applying the policy. If not, the webhook will block the request.
```

Comment:
Which VFs, the ones used by the Host?
And which policy, please clarify we are specifically referring to the SriovNetworkNodePolicy resource
And finally, which webhook are we talking about and which request will it block?